CMakeLists.txt

```diff
@@ -97,6 +101,22 @@ include(SetRpath)
 include(DeclareMod2CSubModule)
 include(FindPkgConfig)
 include(CTest)
+include(Findnmodl)
+
+# ispc support
+if (ENABLE_ISPC_TARGET)
+    enable_language(ISPC)
+    add_definitions("-DISPC_INTEROP=1")
+    set(ENABLE_NMODL ON)
+endif()
+
+if (ENABLE_NMODL)
+    find_package(nmodl REQUIRED)
```

Comment:
What is there both `include(Findnmodl)` and `find_package(nmodl REQUIRED)` ?
Shouldn't the first one be removed?

Reply:
my understanding is that you need to first include the Findnmodl.cmake file such that you can do the find_package command for nmodl...